templates/email/default/alert-update.html

```diff
@@ -13,6 +13,9 @@
 <th style="[% td_style %][% primary_column_style %]" id="primary_column">
   [% start_padded_box | safe %]
   <h1 style="[% h1_style %]">New updates on <a href="[% problem_url %]">[% title %]</a></h1>
+  [% IF cobrand.moniker == 'kingston' AND report.category == 'Request new container' %]
+  <a style="[% button_style %]" href="https://kingston-self.test.achieveservice.com/service/Status_update_for_waste_container_request?client_ref=RBK-[% report.id %]">Track your request</a>
```

Comment:
As discussed, I'd replace the hardcoded URL in the places it's used with `cobrand.feature('waste_features').track_request_url` or similar so we can change it.